optimum/exporters/onnx/convert.py

```diff
@@ -149,7 +141,7 @@ def validate_model_outputs(
             The path to the exported model.
         onnx_named_outputs (`List[str]`):
             The names of the outputs to check.
-        atol (`float`):
+        atol (`float`, defaults to `None`):
```

Comment:
```suggestion
        atol (`Optional[float]`, defaults to `None`):
```

Aligning with https://huggingface.slack.com/archives/C02P0559X9S/p1669628754896019